File: EmployeeManager/EmployeeManager.Shared/DTOs/EmployeeDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace EmployeeManager.Shared.DTOs
{
    public class EmployeeDTO
    {
        public string UserID { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string Email { get; set; }
        public string PhoneNum { get; set; }
        public DateTime DOB { get; set; }
        public bool isDeleted { get; set; }
    }
}

File: EmployeeManager/employeemanager.client/src/models/employeeDTO.ts
export interface EmployeeDTO {

    userID: string,
    firstName: string,
    lastName: string,
    email: string,
    phoneNum: string,
    DOB: Date,
    isDeleted: boolean,
    addresss: AddressDTO,
    skills: SkillsDTO[],
}

interface AddressDTO {

    userID: string;
    streetAddress: string;
    city: string;
    postalCode: string;
    country: string;
}

interface SkillsDTO {

    userID: string;
    skillID: string;
    name: string;
    yearsExperience: number;
    seniority: string;
}